mautic/mautic
# remove unrelevant filters for dynamic content

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Dynamic content support just match results based on contact/company fields https://github.com/mautic/mautic/blob/staging/app/bundles/EmailBundle/EventListener/MatchFilterForLeadTrait.php#L21
Then there is not reason to use filters like Visited URL, Page hit, Email Read, Email sent etc. This PR allow this extended filters just for segments, where Mautic support it.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Try create segments  - filters should contains all filter options 

![image](https://user-images.githubusercontent.com/462477/94560368-d778c980-0262-11eb-9d69-84567dde1db9.png)

3. Try create email and dynamic content - filters should contains just contact/company filter options
